torch-sys/libtch/torch_api.cpp

```diff
@@ -925,6 +925,18 @@ int atc_cudnn_is_available() {
   return -1;
 }
 
+void atc_manual_seed(int64_t seed) {
```

Comment:
Shouldn't this be an uint rather than an int?

Reply:
Yes, will fix that